appveyor/install_zstd.cmd

```diff
@@ -0,0 +1,10 @@
+REM ========================================================
+REM Install zstd
+curl -fsSL -o zstd-v1.4.4.zip https://github.com/facebook/zstd/archive/v1.4.4.zip || exit /b 1
+7z x zstd-v1.4.4.zip || exit /b 1
+cd zstd-1.4.4
```

Comment:
The CI on windows fails.
We should cd to `zstd-1.4.4/build/meson` as we do on kiwix-build side.

Reply:
My fault. Corrected.